moveit_ros/planning/rdf_loader/src/rdf_loader.cpp

```diff
@@ -160,25 +168,25 @@ bool rdf_loader::RDFLoader::loadXacroFileToString(std::string& buffer, const std
 {
   if (path.empty())
   {
-    ROS_ERROR_NAMED("rdf_loader", "Path is empty");
+    RCLCPP_ERROR(LOGGER, "Path is empty");
     return false;
   }
 
   if (!boost::filesystem::exists(path))
   {
-    ROS_ERROR_NAMED("rdf_loader", "File does not exist");
+    RCLCPP_ERROR(LOGGER, "File does not exist");
     return false;
   }
-
-  std::string cmd = "rosrun xacro xacro ";
+  //TODO (anasarrak): Test ros2 xacro https://github.com/ros/xacro/tree/ros2 // https://github.com/bponsler/xacro
+  std::string cmd = "ros2 run xacro xacro";
```

Comment:
Do we have a demonstration of the RDF Loader loading it loading a robot description? Seems like a test that we should really consider adding